en/docs/insights/choreo-insights-api.md

```diff
@@ -0,0 +1,4214 @@
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>apiIds</strong></td>
+<td valign="top">[<a href="#string">String</a>]</td>
+<td>
+
+List of API Ids where the results needs to be filtered with. The maximum number of API Ids that can be defined is
+limited to 5. This field is optional. Hence, if there is no need of filtering by this attribute, omit the field.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>granularity</strong></td>
+<td valign="top"><a href="#string">String</a></td>
+<td>
+
+Granularity value which will be used in data retrieval. This field is optional. Hence, if granularity value is not
+set, default granularity for the related time range will be applied. Granularity value must be one of '1m', '15m',
+'1h', '1d' or '7d' values.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### APIUsageOverTimeFilter
+
+Filters API usage overtime results.
+
+<table>
+<thead>
+<tr>
+<th colspan="2" align="left">Field</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>apiIds</strong></td>
+<td valign="top">[<a href="#string">String</a>]</td>
+<td>
+
+List of API Ids where the results needs to be filtered with. The maximum number of API Ids that can be defined is
+limited to 5. This field is optional. Hence, if there is no need of filtering by this attribute, omit the field.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>appIds</strong></td>
+<td valign="top">[<a href="#string">String</a>]</td>
+<td>
+
+List of application Ids where the results needs to be filtered with. The maximum number of application Ids that can
+be defined is limited to 5. This field is optional. Hence, if there is no need of filtering by this attribute, omit
+the field.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>granularity</strong></td>
+<td valign="top"><a href="#string">String</a></td>
+<td>
+
+Granularity value which will be used in data retrieval. This field is optional. Hence, if granularity value is not
+set, default granularity for the related time range will be applied. Granularity value must be one of '1m', '15m',
+'1h', '1d' or '7d' values.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### AlertSummaryFilter
+
+Filters alerts summary results.
+
+<table>
+<thead>
+<tr>
+<th colspan="2" align="left">Field</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>paginationFilter</strong></td>
+<td valign="top"><a href="#paginationfilter">PaginationFilter</a></td>
+<td>
+
+Pagination filter. This field is optional. Hence, if there is no need of filtering by this attribute, omit the field
+.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>searchFilter</strong></td>
+<td valign="top"><a href="#searchfilter">SearchFilter</a></td>
+<td>
+
+Search filter. This field is optional. Hence, if there is no need of filtering by this attribute, omit the field.
```

Comment:
```suggestion
The search filter. This is an optional parameter. Therefore, if there is no requirement to filter by this attribute, omit this parameter.
```

Reply:
Is "attribute" and "field" the same? We might need to revisit this.

Reply:
yes